AUBGTheHUB/monolith
# #520 Make questionnaire responsive

resolves #520  

## What has been done
Reworked the responsive css logic in the questionnaire so that it looks good on all devices

## How to verify
Run the questionnaire locally and verify how it looks by opening the website in inspect, select responsive and experiment with different resolutions
